exporter/newrelicexporter/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/newrelicexporter
+
+go 1.14
+
+require (
+	github.com/census-instrumentation/opencensus-proto v0.2.1
+	github.com/golang/protobuf v1.4.1
+	github.com/newrelic/newrelic-telemetry-sdk-go v0.2.0
+	github.com/stretchr/testify v1.5.1
+	go.opencensus.io v0.22.3
+	go.opentelemetry.io/collector v0.3.1-0.20200526215338-87af38a24676
+	go.uber.org/zap v1.10.0
+	google.golang.org/protobuf v1.24.0
```

Comment:
I remember seeing this as being a problem for performance. You may want to use 1.23.5 I think is the version we used everywhere.

Reply:
Downgraded to 1.23.0 (https://pkg.go.dev/mod/google.golang.org/protobuf@v1.24.0?tab=versions)